javascript/javascript1/week2/weather.js

```diff
@@ -0,0 +1,21 @@
+weatherWear = (temperature) => {
+  if (temperature < -50 || temperature > 50) {
+    return "Error,temperature is unreal";
+  } else if (temperature < 0) {
+    return "a warm down jacket";
+  } else if (temperature >= 0 && temperature <= 15) {
+    return " a jacket";
+  } else if (temperature >= 16 && temperature <= 20) {
+    return " a light jacket";
```

Comment:
Did you test these? 
I think both conditions(0 and 15) and also(16 and 20) have to pass before the person has to wear a jacket or?

I think it should be like how you made this `if (temperature < -50 || temperature > 50) `

Reply:
from 0 to 15 is jacket, and for 16-20 is a LIGHT jacket, no?

Reply:
Okay, that makes sense. :)